### ISSUE TITLE
new session filter: non-empty narrative

### ISSUE THREAD

**Issue by nsteinme** [cortex-lab/alyx#642]
Would be great to have a filter in the admin interface that returns sessions whose narrative is not empty and it is not equal to "auto-generated session". It should also return sessions that have attached "Notes". Otherwise it is difficult to find where you put your notes when there are multiple sessions per subject per day.

**Comment by k1o0** [comment]
Agreed. We need to make it easier for people to use Alyx for lab notes. The following would be helpful:

- [ ] Promote Notes list view to 'Common' section
- [ ] Add filterset for Notes incl. content object type, users, date range and whether it includes an image
- [ ] Add 'has note' and 'has narrative' filters to Session list view
